server/models/user.model.js

```diff
@@ -0,0 +1,111 @@
+const mongoose = require('mongoose');
+const FoodSchema = require('mongoose').model('Food').schema;
+
+const UserSchema = new mongoose.Schema({
```

Comment:
pls edit current UserSchema

Reply:
I used the current UserSchema to update, just renamed user.js to user.model.js
Should we rename our files because I see many files in folders having the same name user.js. It made me confused and hard to track